src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+  &__item {
+    margin-right: 20px;
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+  &__link {
+    position: relative;
+
+    display: flex;
+    padding: 23px 0;
+
+    font-size: 12px;
+    font-weight: 500;
```

Comment:
Let`s create variables for all properties like font-weight, font-size, etc. It will be more useful in the future

Reply:
ok, fix